mvvmfx/src/main/java/de/saxsys/mvvmfx/internal/viewloader/ViewLoaderReflectionUtils.java

```diff
@@ -397,19 +399,23 @@ static Object injectScopeIntoField(Field scopeField, Object viewModel, ContextIm
                         "https://github.com/sialcasa/mvvmFX/wiki/Dependency-Injection#lifecycle-postconstruct", viewModel));
             }
 
-            AccessController.doPrivileged((PrivilegedAction) () -> {
-                try {
-                    return initMethod.invoke(viewModel);
-                } catch (InvocationTargetException | IllegalAccessException e) {
-                    throw new IllegalStateException(
-                            "mvvmFX wasn't able to call the initialize method of ViewModel [" + viewModel + "].", e);
-                }
-            });
+            AccessController.doPrivileged((PrivilegedAction<?>) () ->
+                    ReflectionUtils.accessMember(initMethod, () -> initMethod.invoke(viewModel), "mvvmFX wasn't able to call the initialize method of ViewModel [" + viewModel + "]."));
```

Comment:
We don't need the `AccessController.doPrivileged` call anymore because this is now done inside of `ReflectionUtils.accessMember`. 
Just `ReflectionUtils.accessMember(initMethod, () -> initMethod.invoke(viewModel), "mvvmFX wasn't able to call the initialize method of ViewModel [" + viewModel + "].");` should work.